docs/design/asset-canister-interface.md

```diff
@@ -0,0 +1,508 @@
+
+The returned `evidence` value must be passed to the `commit_proposed_batch` method.
+
+Required permission: [Prepare](#permission-prepare)
+
+### Method: `commit_proposed_batch`
+
+This method executes the operations previously supplied by [propose_commit_batch()](#method-propose_commit_batch), and deletes the batch.
+
+Preconditions:
+- The batch exists.
+- The batch has proposed commit batch arguments.
+- Evidence computation has completed.
+- Evidence passed in the arguments matches the evidence previously computed by [compute_evidence()](#method-compute_evidence).
+
+Required permission: [Commit](#permission-commit)
+
+### Method: `grant_permission`
+
+This method grants a permission to a principal.
+
+Callable by: Principals with [ManagePermissions](#permission-managepermissions) permission, and canister controllers.
+
+### Method: `revoke_permission`
+
+This method revokes a permission from a principal.
+
+Callable by: Principals with [ManagePermissions](#permission-managepermissions) permission, and canister controllers. Also, any principal can revoke any of its own permissions.
+
+### Method: `list_permitted`
+
+This method returns a list of principals that have the given permission.
+
+### Method: `authorize`
+
+> **NOTE**: This method is deprecated. Use [grant_permission()](#method-grant_permission) instead.
+
+This method grants the [Commit](#Permission_commit) permission to a principal.
+
+Callable by: Principals with [ManagePermissions](#permission-managepermissions) permission, and canister controllers.
+
+### Method: `deauthorize`
+
+> **NOTE**: This method is deprecated. Use [revoke_permission()](#method-revoke_permission) instead.
+
+This method revokes the [Commit](#Permission_commit) permission from a principal.
+
+Callable by: Principals with [ManagePermissions](#permission-managepermissions) permission, and canister controllers. Also, any principal can revoke anny of its own permissions.
+
+### Method: `list_authorized`
+
+> **NOTE**: This method is deprecated. Use [list_permitted()](#method-list_permitted) instead.
+
+This method returns a list of principals that have the [Commit](#Permission_commit) permission.
+
+### Method: `configure`
+
+This method configures the asset canister. See [Configuration Reference](#configuration-reference) for details.
+
+### Method: `get_configuration`
+
+This method returns the configuration of the asset canister.
+
+### Method: `get_asset_properties`
+
+This method returns the properties of the asset with the given key.
+
+### Method: `certified_tree`
+
+This method returns the certified tree.
+
+### Convenience Methods
+
+Each of these methods is the equivalent of its respective batch operation.
+
+While they are provided for "convenience," some don't actually make sense.  For example, [set_asset_content()](#operation-setassetcontent) requires chunk ids, but [create_chunk()](#method-createchunk) requires a batch.
```

Comment:
putting the next two paragraphs within an admonition may be a nice way to add the authorial perspective